providers/ndb.rb

```diff
@@ -50,6 +50,21 @@
   EOF
   end
 
+
+  if node['ndb']['nvme']['logfile_size'] != ""
+
+    bash 'add_disk_data_files' do
```

Comment:
This is not the correct place for this code. For starter the `ndb` provider is called for every nn/rm deployed. Secondly you are trying to touch files in directories that exist only in the ndb datanode machines. The ndb datanodes are not co-located with rms/nns.

Also, there is a python script templated in `ndb-chef` maybe we can avoid code duplication and call that script instead.

Reply:
You're right about the touch file. The problem here is that we want to be able to configure the amount of space available for small files in NDB (on-disk data). I looked around for where it was setup, and i thought it was in the metadata-dal-impl, but if it's in ndb-chef i will move it there.

Reply:
there is a python script, manage_disk_data.py, but it isn't called during installation. I will talk to Mahmoud.